expandedTree/substitutionMatrix_test.go

```diff
@@ -68,19 +68,19 @@ var PopulateSubstitutionMatricesTests = []struct {
 				{0.0214, 0.0214, 0.936, 0.0214},
 				{0.0214, 0.0214, 0.0214, 0.936},
 			},
-			"firstRight": [][]float64{
```

Comment:
Not needed because `[][]float64` is already declared when you initialized the map, see line53:
https://github.com/vertgenlab/gonomics/blob/45e57c55e07141c6f274bbea365097ae24a8753e/expandedTree/substitutionMatrix_test.go#L53